pom.xml

```diff
@@ -635,6 +635,52 @@
                     <generateGitPropertiesFilename>${project.build.outputDirectory}/git.properties</generateGitPropertiesFilename>
                 </configuration>
             </plugin>
+
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>exec-maven-plugin</artifactId>
+                <version>${exec-maven-plugin.version}</version>
+                <executions>
+                    <execution>
+                        <id>create-licenses</id>
+                        <configuration>
+                            <mainClass>io.confluent.licenses.LicenseFinder</mainClass>
+                            <arguments>
+                                <!-- Note use of development instead of package so we pick up all dependencies.
+                                     This assumes both subprojects will be packaged together. -->
+                                <argument>-i ${project.build.directory}/${project.build.finalName}-package/share/java/acl</argument>
+                                <argument>-o ${project.parent.basedir}/licenses/overrides</argument>
```

Comment:
If this is in the root pom, `project.parent.basedir` won't work, you'd just want `project.basedir` since the licenses/override directory is at the root directory as well.

But I think from a quick glance this is in the wrong place anyway. You want it in any pom file where you are using maven-assembly plugin to generate packages:
* -i is the directory that the packaged version lands in, we run assembly plugin then use those jars to generate license info
* You can keep the `parent` part for -o if your overrides are in the root and reused by all components.
* -h -l -n -t should all be fine as is I think
* -x is an ignore flag, it is intended to list any jars your project produces since we don't need to include anything for them -- they are already covered just by the LICENSE file we ship